projects/packages/licensing/src/class-licensing.php

```diff
@@ -269,4 +273,381 @@ public function get_license_activation_notice_dismiss() {
 
 		return $default;
 	}
+
+	/**
+	 * Declare the endpoints for the licensing package.
+	 *
+	 * @since 10.9.0
+	 */
+	public function register_endpoints() {
```

Comment:
What do you think about moving the endpoints to their own class, kinda like we do for the Connection package?
https://github.com/Automattic/jetpack/blob/4a9a5c19ffa1ad972c6e4adbf08e04173d8401b1/projects/packages/connection/src/class-rest-connector.php#L19-L22
https://github.com/Automattic/jetpack/blob/60ff8e8f3400ab8e8bf7c68f1377e506d6f1cfd9/projects/packages/connection/src/class-manager.php#L100